<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>845</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListView" name="listFiles">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>591</width>
      <height>741</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddFile">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>280</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #4CAF50;  /* Green */
    color: white;
    border-radius: 8px;
    padding: 8px;
}

QPushButton:hover {
    background-color: #66BB6A;
}
</string>
    </property>
    <property name="text">
     <string>Upload</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDeleteFile">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>350</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #E53935;
    color: white;
    border-radius: 8px;
    padding: 8px;
}

QPushButton:hover {
    background-color: #EF5350;
}
</string>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOpen">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>200</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #2196F3;  /* Blue */
    color: white;
    border-radius: 8px;
    padding: 8px;
}

QPushButton:hover {
    background-color: #42A5F5;
}
</string>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>845</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
